class.js

```diff
@@ -0,0 +1,86 @@
+const addBtn = document.querySelector('#add-btn');
+const inputTitle = document.querySelector('#input-title');
+const inputAuthor = document.querySelector('#input-author');
+const form = document.querySelector('#form');
+const booksContainer = document.getElementById('books-inner-container');
+const removeBtn = document.querySelector('.remove-book');
+
+class Book {
```

Comment:
- [ ] Removing entries is not working as intended. When the remove button is clicked and the page is refreshed, the removed data shows again. The data appears not to be properly deleted from the local storage when the remove function is invoked. Kindly make sure that the refactoring did not introduce bugs to the app and that all functions from the previous project are still working fine.